packages/execution-environments/src/web-workers/index.ts

```diff
@@ -5,4 +5,4 @@ import { WebWorkerSnapExecutor } from './WebWorkerSnapExecutor';
 executeLockdown();
 executeLockdownMore();
 
-WebWorkerSnapExecutor.initalize();
+WebWorkerSnapExecutor.initialize();
```

Comment:
There was a typo in `initialize` before. Not sure if this is a breaking change, or if this is in use?

Reply:
Technically a breaking change. But only in use for our tests currently.